source/_integrations/zha.markdown

```diff
@@ -265,7 +265,18 @@ custom_quirks_path:
 
 #### Defining Zigbee channel to use
 
-Tip! Before considering to change to an other Zigbee channel on an existing Zigbee network, it is highly recommended that you read through the two segments under the [troubleshooting](#troubleshooting) section below about _Best practices to avoid pairing/connection difficulties_ and _Zigbee interference avoidance and network range/coverage optimization_. These sections provide prerequisite information and advice on how to achieve the best possible Zigbee network in your environment.
+{% warning %}
+The best practice is to **not change the Zigbee channel** from the ZHA default.
+{% endwarning %}
```

Comment:
```suggestion
{% endimportant %}
```
I would be careful with the term "warning", is it collides with the ANSI Z.535 definition for signal words to use in documentation. For safety information, Warning is used to " identify a hazard that "may" cause death and will cause injury".